doc/source/extending.rst

```diff
@@ -113,21 +113,67 @@ by some other storage type, like Python lists.
 See the `extension array source`_ for the interface definition. The docstrings
 and comments contain guidance for properly implementing the interface.
 
+.. _extending.extension.operator:
+
 :class:`~pandas.api.extensions.ExtensionArray` Operator Support
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 By default, there are no operators defined for the class :class:`~pandas.api.extensions.ExtensionArray`.
-There are two ways that you can provide operator support for your ExtensionArray. One
-is to define each of the operators on your ExtensionArray subclass. The second method
-assumes that the underlying elements of the ExtensionArray have the individual operators
-already defined.  An ``ExtensionArray`` subclass consisting of objects that have arithmetic and
-comparison operators defined on the underlying objects can easily support 
-those operators on the ``ExtensionArray``, and therefore the operators 
-on ``Series`` built on those ``ExtensionArray`` classes will work as expected.
+There are two approaches for providing operator support for your ExtensionArray:
+
+1. Define each of the operators on your ExtensionArray subclass. 
+2. Use operators from pandas defined on the ExtensionArray subclass based on already defined
+   operators on the underlying elements.
+    
+For the first approach, you will need to create a mixin class with a single class method,
+with the following signature:
+
+.. code-block:: python
+
+    @classmethod
+    def _create_method(cls, op, coerce_to_dtype=True):
+
+The method ``create_method`` should return a method with the signature
+``binop(self, other)`` that returns the result of applying the operator ``op``
+to your ExtensionArray subclass.  Your mixin class will then become a base class
+for the provided :class:`ExtensionArithmeticOpsMixin` and
+:class:`ExtensionComparisonOpsMixin` classes. 
+
+For example, if your ExtensionArray subclass
+is called ``MyExtensionArray``, you could create a mixin class ``MyOpsMixin``
+that has the following skeleton:
+
+.. code-block:: python
+
+    class MyOpsMixin(object):
+    @classmethod
+    def _create_method(cls, op, coerce_to_dtype=True):
```

Comment:
extra level of indentation for valid python code

Reply:
fixed